swift/Picture.swift

```diff
@@ -0,0 +1,72 @@
+import CoreGraphics
+
+import thorvg
+
+/// A Swift wrapper for ThorVG's Picture, facilitating picture manipulation.
+class Picture {
+    /// Supported MIME types for loading picture data.
+    enum MimeType: String {
+        case svg = "svg"
+        case lottie = "lottie"
+        case jpg = "jpg"
+        case jpeg = "jpeg"
+        case svgxml = "svg+xml"
+        case png = "png"
```

Comment:
Does this package support SVG and images as well as Lottie?
Otherwise, we can remove them to avoid misunderstanding.